packages/expo-dev-launcher/bundle/functions/getDevSessionsAsync.ts

```diff
@@ -29,7 +29,7 @@ export async function getDevSessionsAsync({
     devSessions = devSessions.concat(sessions);
   }
 
-  if (!devSessions.length && !isDevice) {
```

Comment:
in light of our conversation earlier -- was this an intentional change?

Reply:
nope totally missed this - thanks for flagging I'll revert